lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2147,6 +2151,13 @@ definitions:
           - 'HEX2BIN'
         example: 'BASE64'
 
+  subscriberdb_sync_interval:
+    description: Interval in seconds between gateway requests to sync its subscriberdb with the cloud.
+    type: integer
+    format: uint32
+    x-nullable: true
```

Comment:
This is the default I believe, so can omit it

Reply:
I looked into this and it seems `nullable` is an interesting thing in go-swagger: https://goswagger.io/use/models/schemas.html

It appears that `x-nullable: false` is the default. `x-nullable: true` turns basic data types into pointers with `nil` values when the field is unset, whereas by default, the type would hold empty values such as `0` and `""`. Keeping `x-nullable: false` by omitting it means that the value would be `0` when not set. It won't be required. I think we are okay with that in this case and can treat `0` as unset.